src/app/[locale]/rest-client/page.tsx

```diff
@@ -27,115 +27,97 @@ export default function RestClientPage() {
     setError(null);
     setResponse(null);
 
+    if (values.contentType === ContentType.JSON && values.data?.trim()) {
```

Comment:
If you destructurize as indicated in the comment to the file
```suggestion
    if (contentType === ContentType.JSON && data?.trim()) {
```

Reply:
I have done it like that, thank you!